spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/main/java/org/springframework/cloud/gcp/sql/autoconfig/GcpCloudSqlAutoConfiguration.java

```diff
@@ -75,44 +76,36 @@ protected SQLAdmin sqlAdminService(CredentialsProvider credentialsProvider)
 
 		return new SQLAdmin.Builder(httpTransport, jsonFactory,
 				new HttpCredentialsAdapter(credentialsProvider.getCredentials()))
-				.build();
+						.build();
```

Comment:
I think there is just no rule on the matter and it even can't be possible IMO.

OTOH this looks better than previous one